torchtune/datasets/multimodal/_the_cauldron.py

```diff
@@ -235,3 +235,17 @@ def __call__(self, sample: Mapping[str, Any]) -> Mapping[str, Any]:
     if packed:
         raise ValueError("Multimodal datasets don't support packing yet.")
     return ds
+
+
+def the_cauldron_transform(
```

Comment:
I absolutely love this.

Reply:
Although, we may not need the "column_map" b/c if it's truly just for the cauldron, then we know what the columns will be, no? It would be much clearer IMO to have e.g. image_column, text_column, etc.